app/components/primer/clipboard_copy.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Primer
```

Comment:
Might you be able to add this component to the docs per https://github.com/primer/view_components/blob/main/CONTRIBUTING.md#submitting-a-pull-request?